styles/go.dev/catppuccin.user.css

```
@@ -0,0 +1,938 @@
+/* ==UserStyle==
+@name go.dev Catppuccin
+@namespace github.com/catppuccin/userstyles/styles/go.dev
+@homepageURL https://github.com/catppuccin/userstyles/tree/main/styles/go.dev
+@version 1.0.0
+@updateURL https://github.com/catppuccin/userstyles/raw/main/styles/go.dev/catppuccin.user.css
+@supportURL https://github.com/catppuccin/userstyles/issues?q=is%3Aopen+is%3Aissue+label%3Ago.dev
+@description Soothing pastel theme for go.dev
+@author Catppuccin
+@license MIT
+
+@preprocessor less
+@var select lightFlavor "Light Flavor" ["latte:Latte*", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha"]
+@var select darkFlavor "Dark Flavor" ["latte:Latte", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha*"]
+@var select accentColor "Accent" ["rosewater:Rosewater", "flamingo:Flamingo", "pink:Pink", "mauve:Mauve*", "red:Red", "maroon:Maroon", "peach:Peach", "yellow:Yellow", "green:Green", "teal:Teal", "blue:Blue", "sapphire:Sapphire", "sky:Sky", "lavender:Lavender", "subtext0:Gray"]
+==/UserStyle== */
+
+@-moz-document domain('go.dev') {
+  @media (prefers-color-scheme: light) {
+    :root {
+      #catppuccin(@lightFlavor, @accentColor);
+    }
+  }
+  @media (prefers-color-scheme: dark) {
+    :root:not([data-theme='light']) {
+      #catppuccin(@darkFlavor, @accentColor);
+    }
+  }
```

Can we not do it via `[data-theme='auto']` here?
```suggestion
  @media (prefers-color-scheme: light) {
    :root {
      #catppuccin(@lightFlavor, @accentColor);
    }
  }
  @media (prefers-color-scheme: dark) {
    :root:not([data-theme='light']) {
      #catppuccin(@darkFlavor, @accentColor);
    }
  }
```

---

Honestly, I don't know how I missed that.

---

I could have sworn you *did* have it like this earlier ahah